src/main/java/org/mockbukkit/mockbukkit/inventory/meta/BundleMetaMock.java

```diff
@@ -43,7 +43,7 @@ public BundleMetaMock(@NotNull ItemMeta meta)
 
 		if (meta instanceof BundleMeta bundleMeta)
 		{
-			this.items = new ArrayList<>(bundleMeta.getItems());
+			this.items = new ArrayList<>(bundleMeta.getItems().stream().map(ItemStack::clone).toList());
```

Comment:
Can't comment on the right line, but shouldn't this be :
```
	public BundleMetaMock(@NotNull ItemMeta meta)
	public BundleMetaMock(@NotNull BundleMetaMock meta)
```
To be a correct copy-constructor?

Reply:
That's a great question, i noticed that too. Not sure if that's the main reason, but i saw that that is also happening in PaperMC:
<img width="892" height="512" alt="imagem" src="https://github.com/user-attachments/assets/754ed08a-bc64-4dd8-8092-02b407257b00" />
Maybe it was a replication of this?